### ISSUE TITLE
Shortcut to search in whole library

### ISSUE THREAD

**Issue by e-pet** [JabRef/jabref#10643]
I have a large library with many groups, subgroups, subsubgroups, etc. I will often have some subgroup selected but then want to search for something in the whole library. My current workflow for this requires that I scroll up all the way in the groups panel (which often requires scrolling through multiple screens of groups), manually select "All entries" and then proceed to do my search. This is quite cumbersome.

Is there a quicker way to do this that I am unaware of? If not, any kind of shortcut would be much appreciated. 

Naively, I would think that the best solution might be to make the "All entries" "group" sticky, such that it is always visible at the top of the groups list no matter how far I have scrolled down.

**Comment by ThiloteE** [comment]
Have you tried the "filter groups" feature already?
![image](https://github.com/JabRef/jabref/assets/73715071/3e79a11b-8474-44ae-a236-8bc34fa9b450)

**Comment by e-pet** [comment]
Yes… you’re right; that is possibly slightly faster than scrolling up. Still not very convenient though for a core workflow element (for me, at least). (Click in the filter groups field – manually type “All ..” – select the group – click in search field – type actual search entry.)

Also, if I then want to go back to the group I was currently in / working on, I have to manually remove the group filter again, and then either search for it or scroll down all the way again. (If there was a sticky “All groups” at the top, and clicking on that would not change the current “scroll position” of the groups pane, I could immediately and easily get back to where I was before the search.)


Von Outlook<http://aka.ms/weboutlook> gesendet.
